From 1cd978c028711391df0b992780eebe885285156a Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Fri, 17 May 2013 09:25:09 -0700
Subject: [PATCH 96/96] HBASE-8563 Double count of read requests for Gets

git-svn-id: https://svn.apache.org/repos/asf/hbase/branches/0.94@1483796 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 042b9b537bc38e42b8ff7d5c405dbf3436cb32d4)

Reason: Bug
Author: Francis Liu
Ref: CDH-12246
---
 .../apache/hadoop/hbase/regionserver/HRegion.java  |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java b/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
index f3be643..6d9a9b7 100644
--- a/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
+++ b/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
@@ -1742,8 +1742,6 @@ public class HRegion implements HeapSize { // , Writable{
   protected RegionScanner getScanner(Scan scan,
       List<KeyValueScanner> additionalScanners) throws IOException {
     startRegionOperation();
-    this.readRequestsCount.increment();
-    this.opMetrics.setReadRequestCountMetrics(this.readRequestsCount.get());
     try {
       // Verify families are all valid
       prepareScanner(scan);
-- 
1.7.0.4

